pynest/examples/hpc_benchmark.py

```diff
@@ -34,6 +34,21 @@
 
 This is the standard network investigated in [1]_, [2]_, [3]_.
 
+A note on connectivity
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: ../../../examples/hpc_benchmark_connectivity.svg
```

Comment:
This path is something that we need to fix across all examples. This is also different from the Potjans example, but only the filename does not seem to work. Anyway hardcoding the fixed filenames in `conf.py` feels very wrong to me, but requires more of a structural review outside this pr.
I'd say this is fine for the case at hand.